src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upSide;
+    private int downSide;
+    private int height;
+
+    public void setUpSide(int upSide) {
+        this.upSide = upSide;
+    }
+
+    public void setDownSide(int downSide) {
+        this.downSide = downSide;
+    }
+
+    public void setHeight(int height) {
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (this.upSide + this.downSide) * this.height * 0.5;
```

Comment:
```suggestion
        return (upSide + downSide) * height * 0.5;
```

тут нам тоже не нужно использовать слово `this`